[EDU-3453] feat: Suite Bypass template guide (#531)
aziontech/docs: src/content/docs/en/pages/guides/marketplace/templates/by-pass-template/by-pass-template.mdx

---

> title: How to deploy the Suite By Pass template

Author: ```suggestion
title: How to deploy the Suite Bypass template
```

> description: This template includes configurations for adding a layer of engine rules to ignore all cache settings for a specific URL route.

Author: ```suggestion
description: This template includes configurations that add a layer of engine rules to ignore all cache settings for a specific URL route.
```

meta_tags: templates, guides, Azion Marketplace
namespace: docs_guides_by_pass_template

> permalink: /documentation/products/guides/suite-by-pass/

Author: ```suggestion
permalink: /documentation/products/guides/suite-bypass/
```

---

import Badge from '~/components/Badge.astro';

<Badge variant="accent">
Preview
</Badge>

> **Suite Bypass** includes configurations for adding a layer of engine rules to ignore all cache settings for a specific URL route. These configurations use a standard rule based on criteria and behaviors applied through Rules Engine for Edge Application.

Reviewer: ```suggestion
**Suite Bypass** includes configurations that add a layer of engine rules to ignore all cache settings for a specific URL route. These configurations use a standard rule based on criteria and behaviors applied through Rules Engine for Edge Application.
```

> **Suite By Pass** includes configurations for adding a layer of engine rules to ignore all cache settings for a specific URL route. These configurations use a standard rule based on criteria and behaviors applied through Rules Engine for Edge Application.

Author: ```suggestion
**Suite Bypass** includes configurations for adding a layer of engine rules to ignore all cache settings for a specific URL route. These configurations use a standard rule based on criteria and behaviors applied through Rules Engine for Edge Application.
```

The rule is composed by:

- **Criteria**: if `${uri}` *starts with* `/url_route`.
- **Behavior**: then *Set Origin* `Default Origin`, *Bypass cache phase*, and *Forward Cookies*.

You must link your origin with the template to start using this set of rules. After the deployment, the script creates a new edge application that runs on **Azion's Edge Computing Platform**, with which you can manage and adjust the settings. 

---

## Getting the template

To use this template, access [Real-Time Manager (RTM)](https://manager.azion.com/) and select the **Start with a template** option on the homepage.

> This will open a new page where you can select the **Suite By Pass** card.

Author: ```suggestion
This will open a new page where you can select the **Suite Bypass** card.
```

---

## Setting up the template

Once you're in the template's page, click the **Settings** tab to open the configuration form and provide the information to configure your application. 

Fields identified with an asterisk are mandatory.

- **Application Name** *: the name of your edge application on Azion.
    - Use a unique and easy-to-remember name. If the name has already been used, the platform returns an error message.
- **Origin Address** *: the address of the hosted origin the edge application should fetch the original content from.
    - Example: `www.yourwebsite.com`
- **Custom Host Header**: in some cases, it may be necessary to add host information in the header request to retrieve information from the origin. This variable is used for that purpose.
    - Example: `${host}`. Read more about the host header and how to customize it on the [Origins documentation](/en/documentation/products/edge-application/origins/#host-header).

> - **Route to bypass**: specific URL route to apply the bypassing rules.

Author: ```suggestion
- **Route to bypass** *: specific URL route to apply the bypassing rules.
```

Additionally, to guarantee the optimal performance of this template, you must activate:

<Badge>
[Edge Functions](/en/documentation/products/edge-application/edge-functions/)
</Badge>
<Badge>
[Application Acceleration](/en/documentation/products/edge-application/application-acceleration/)
</Badge>

You must activate these modules separately via RTM. To do so:

1. [Access RTM](https://manager.azion.com/).
2. On the upper-left corner, select the three horizontal lines to open the **Products menu**.
3. In the **BUILD** section, select **Edge Application**.
4. You'll be redirected to the **Edge Application** page.
- It lists all the edge applications you've created. 
5. Find the edge application related to your template and select it. 
6. In the **Main Settings** tab, find the **Edge Application Modules** section and activate the switch for the product you want to enable.

:::caution[Warning]
If these modules are activated, the execution of this template could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information. 
:::

---

## Managing the template

> Considering that this initial setup may not be optimal for your specific edge application, all settings can be customized any time you need by using RTM.

Reviewer 2: ```suggestion
Considering that this initial setup may not be optimal for your specific edge application, all settings can be customized at any time by using RTM.
```

Reviewer 2: acho que essa frase se repete em todos os guias das integrations, mas foi uma sugestão que me veio agora hehe
